src/dune_rules/module_compilation.ml

```diff
@@ -199,6 +194,21 @@ let build_cm cctx ~precompiled_cmi ~cm_kind (m : Module.t)
     |> List.concat_map ~f:(fun p ->
            [ Command.Args.A "-I"; Path (Path.build p) ])
   in
+  let melange_args =
+    match cm_kind with
+    | Melange Cmj ->
+      let pkg_name_args =
+        match CC.package cctx with
+        | None -> []
+        | Some pkg ->
+          [ "--bs-package-name"; Package.Name.to_string (Package.name pkg) ]
+      in
+      "--bs-stop-after-cmj" :: "--bs-package-output"
+      :: (* This should prob be Path.t or Path.Source.t *)
```

Comment:
I think this comment is wrong, how can it ever be a source path?

Reply:
Yeah source path doesn't make sense, but what I wanted is a `Path.t` :) removed the comment in https://github.com/ocaml/dune/pull/6349#discussion_r1008689179 as it doesn't apply anymore.